frontend/packages/kubevirt-plugin/integration-tests/tests/vm.list-filtering.scenario.ts

```diff
@@ -0,0 +1,40 @@
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { createResource, deleteResource } from '@console/shared/src/test-utils/utils';
+import { getVMManifest } from './utils/mocks';
+import { VM_STATUS, VM_ACTION } from './utils/consts';
+import { VirtualMachine } from './models/virtualMachine';
+import { filterBoxCount } from '../views/vms.list.view';
+
+describe('Test List View Filtering', () => {
+  const testVM = getVMManifest('URL', testName);
+  const vm = new VirtualMachine(testVM.metadata);
+
+  beforeAll(async () => {
+    createResource(testVM);
+  });
+
+  afterAll(() => {
+    deleteResource(testVM);
+  });
+
+  it('Displays correct count of Other VMs', async () => {
+    await vm.waitForStatus(VM_STATUS.Importing);
+    await vm.navigateToListView();
+    const importingCount = await filterBoxCount('Importing');
```

Comment:
can this function return number already so we don't have to do the conversion each time we call it?

Reply:
Yes, updated.